include/picongpu/particles/pusher/particlePusherHigueraCary.hpp

```diff
@@ -99,7 +99,7 @@ namespace picongpu
                 sqrt_HC::float3_X const tau
                     = precisionCast<sqrt_HC::float_X>(float_X(0.5) * bField * charge * deltaT / mass);
 
-                sqrt_HC::float_X const sigma = pmacc::math::abs2(gamma_minus) - pmacc::math::abs2(tau);
+                sqrt_HC::float_X const sigma = math::abs(gamma_minus) - pmacc::math::l2norm2(tau);
```

Comment:
that does not look right:
`abs` is `|x|` but before it used to be `|x|^2` - should this be (C++ like) `norm` here? 
(the second replacement `abs2` to `l2norm2` is fine.

Reply:
well spotted, yes `abs` is wrong here